### PR TITLE
Add Calligraphy and uni sans typeface

### PR DESCRIPTION
**Pull request by caro5**
@sukhada

### CODE REVIEW

File: app/src/main/java/turtler/voyageur/adapters/EventAdapter.java [hunk 1]
```diff
@@ -113,15 +111,18 @@ public void onBindViewHolder(EventAdapter.ViewHolder viewHolder, int position) {
...
         }
-
-        viewHolder.tvDate.setText(TimeFormatUtils.dateTimeToString(ev.getDate()));
-        viewHolder.tvCaption.setText(ev.getCaption());
-        viewHolder.tvTitle.setText(ev.getTitle());
-
+        if (ev.getDate() != null) {
+            viewHolder.tvDate.setText(TimeFormatUtils.dateTimeToString(ev.getDate()));
+        }
+        if (ev.getDate() != null) {
```
Comment (sukhada) [inline app/src/main/java/turtler/voyageur/adapters/EventAdapter.java]: shouldn't this be `ev.getCaption()`?

File: app/src/main/java/turtler/voyageur/adapters/EventAdapter.java [hunk 2]
```diff
@@ -113,15 +111,18 @@ public void onBindViewHolder(EventAdapter.ViewHolder viewHolder, int position) {
...
+            viewHolder.tvCaption.setText(ev.getCaption());
+        }
+        if (ev.getDate() != null) {
```
Comment (sukhada) [inline app/src/main/java/turtler/voyageur/adapters/EventAdapter.java]: `ev.getTitle()`?

File: app/src/main/java/turtler/voyageur/adapters/EventAdapter.java (hunk 1 above)
Comment (caro5) [reply app/src/main/java/turtler/voyageur/adapters/EventAdapter.java]: lol oops

**Comment by caro5** [comment]
@sukhada updated.

**Pull request merged**